### PR TITLE
Fix idempotency in apt module

### PR DESCRIPTION
**Pull request by jjshoe**
Only show a change, if we did indeed update the cache.

### CODE REVIEW

File: lib/ansible/modules/packaging/os/apt.py [hunk 1]
```diff
@@ -1017,7 +1017,10 @@ def main():
                 else:
                     module.fail_json(msg='Failed to update apt cache: %s' % err)
                 cache.open(progress=None)
-                updated_cache = True
+                mtimestamp, post_cache_update_time = get_updated_cache_time()
+                if updated_cache_time != post_cache_update_time:
+                    updated_cache = True
+                    updated_cache_time = post_cache_update_time
                 mtimestamp, updated_cache_time = get_updated_cache_time()
```
Comment (mkrizek) [inline lib/ansible/modules/packaging/os/apt.py]: I think this call is now redundant, no?

Comment (jjshoe) [reply lib/ansible/modules/packaging/os/apt.py]: Yes, and no, right now it is "correct". The call on L1020 sets a different variable name to be used in the compare. 

I should probably just indent back 1023, and delete 1024.

**Comment by jjshoe** [comment]
@mkrizek - Made a change if you feel like re-reviewing.

**Review by dagwieers — APPROVED** [review]
LGTM

**Pull request merged**